src/lib/utils.js

```diff
@@ -68,7 +68,7 @@ let utils = {
       if (item[property] === undefined) {
         throw new Error(`[groupBy]: Object has no key ${property}`);
       }
-      let key = item[property];
+      let key = item[property] && item[property].toLowerCase();
```

Comment:
Then we should update the `if` a few line above, where it is testing if it's `undefined` to also test `null`